.github/workflows/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+**Bug description**
+A clear and concise description of what the bug is. Please do not forget to add:
+- Orion version: X.Y.Z (this can be get with `GET /version` in the API port or using `contextBroker --version`)- 
+MongoDB version X.Y
```

Comment:
```suggestion
- Orion version: X.Y.Z (this can be got with `GET /version` in the API port or using `contextBroker --version`)
- MongoDB version X.Y
```